vector/src/main/res/layout/activity_bug_report.xml

```diff
@@ -154,7 +154,7 @@
                     android:layout_height="wrap_content"
                     android:layout_marginStart="10dp"
                     android:layout_marginEnd="10dp"
-                    android:checked="true"
+                    android:checked="false"
```

Comment:
This toggle also affects the visibility of the screenshot on this screen (`bug_report_screenshot_preview`).
Maybe hide the screenshot by default, or make it always visible? But maybe stay like that, so the user can see the screenshot that will be sent if they check the box, and when they check and uncheck, the screenshot will be GONE.
Also, if the checkbox is not checked, and the user sends the report, maybe add a dialog like "Do you want to include a screenshot to your report?". I am afraid that we will get less screenshots with this change, even if generally there are quite useless.

Reply:
:+1:  for making the screenshot preview always visible  https://github.com/vector-im/element-android/pull/6297/commits/d8814974c56550ca96cb77c44b8bd368adecb2bb

for the UX change this feels like something that would need product input, most bug reports which include screenshots often don't contain the reported bug (eg crashes)